Src/AmrCore/AMReX_MFInterpolater.cpp

```diff
@@ -202,7 +204,9 @@ MFCellConsLinInterp::interp (MultiFab const& crsemf, int ccomp, MultiFab& finemf
             });
         }
 
-        Gpu::streamSynchronize();
```

Comment:
I think we need to keep this one because there is a local MulitiFab in this block.

Reply:
I think it's better to keep the stream sync so that the async arena does not use too much memory.